sematic/ui/src/components/RunPanel.tsx

```diff
@@ -152,3 +132,50 @@ export default function RunPanel(props: {
     </Box>
   );
 }
+
+function RunActionMenu(props: {
+  run: Run;
+  edges: Edge[];
+  resolution: Resolution;
+}) {
+  const { run, edges, resolution } = props;
+
+  const { user } = useContext(UserContext);
+
+  const onRerunClick = useCallback(() => {
+    fetchJSON({
+      url: "/api/v1/resolutions/" + run.root_id + "/rerun",
+      method: "POST",
+      body: { rerun_from: run.id },
+      apiKey: user?.api_key,
+      callback: (payload) => {},
+      setError: (error) => {},
```

Comment:
Please don't silent ignore. This would be a bad experience for the user.

Reply:
Done.